src/components/DeploymentsModal.tsx

```diff
@@ -185,54 +190,76 @@ export const DeploymentsModal = ({ handleCloseModal, isModalOpen }: IDeployments
         variant={ModalVariant.large}
       >
         {deployments.length > 0 ? (
-          <TableComposable aria-label="List of deployments">
-            <Thead>
-              <Tr>
-                <Th sort={getSortParams(0)}>{columnNames.name}</Th>
-                <Th modifier="wrap" info={{ tooltip: 'Cluster namespace' }}>
-                  {columnNames.namespace}
-                </Th>
-                <Th modifier="wrap" sort={getSortParams(2)}>
-                  {columnNames.date}
-                </Th>
-                <Th modifier="wrap">{columnNames.errors}</Th>
-                <Th modifier="wrap">{columnNames.status}</Th>
-                <Th modifier="wrap">{columnNames.type}</Th>
-                <Th></Th>
-              </Tr>
-            </Thead>
-            <Tbody>
-              {sortedDeployments.map((dep, rowIndex) => (
-                <Tr key={rowIndex}>
-                  <Td dataLabel={columnNames.name}>
-                    <b>{dep.name}</b>
-                  </Td>
-                  <Td dataLabel={columnNames.namespace}>{dep.namespace}</Td>
-                  <Td dataLabel={columnNames.date}>{formatDateTime(dep.date)}</Td>
-                  <Td dataLabel={columnNames.errors}>
-                    {dep.errors.length > 0 && (
-                      <span>
-                        <CustomExclamationTriangleIcon color="red" />
-                        &nbsp;&nbsp;
-                      </span>
-                    )}
-                    {dep.errors.length}
-                  </Td>
-                  <Td dataLabel={columnNames.status}>{dep.status}</Td>
-                  <Td dataLabel={columnNames.type}>
-                    <Badge key={rowIndex} isRead>
-                      {dep.type}
-                    </Badge>
-                  </Td>
-                  <Td isActionCell>
-                    <ActionsColumn items={defaultActions(dep)} />
-                  </Td>
+          <>
+            <Toolbar>
+              <ToolbarContent className={'deoployments__toolbarContent'}>
```

Comment:
I think there's a typo in the className

Reply:
but I guess do we also need a classname here? I didn't see this added to the CSS

Reply:
sure, sounds good - removed. Thanks! 
Also removed the className={'table-wrap'} from the TableComposable, as I missed that one too.